.drone.star

```diff
@@ -689,12 +684,11 @@ def accountsUITests(ctx, webBranch, webCommit, storage = 'owncloud', accounts_ha
           'FEATURE_PATH': '/drone/src/accounts/ui/tests/acceptance/features',
         },
         'commands': [
+          'source /drone/src/.drone.env',
           'git clone -b master --depth=1 https://github.com/owncloud/testing.git /srv/app/testing',
-          'git clone -b %s --single-branch --no-tags https://github.com/owncloud/web.git /srv/app/web' % (webBranch),
+          'git clone -b $WEB_BRANCH --single-branch --no-tags https://github.com/owncloud/web.git /srv/app/web',
           'cd /srv/app/web',
-        ] + ([
-          'git checkout %s' % (webCommit)
-        ] if webCommit != '' else []) + [
+          'git checkout $WEB_COMMITID',
```

Comment:
This means that when testing against a branch, we will have to specify a commit id.
But actually, any git reference can go there. So if the person doing the "dummy test" PR puts the branch name in both `WEB_BRANCH` and `WEB_COMMITID` then that will work fine. It will specifically do a `git checkout ` of the branch that was just cloned - which does not hurt.